File: app/src/main/java/suzp1984/github/io/exapidemo/app/fragments/ListAdapter2.java
package suzp1984.github.io.exapidemo.app.fragments;

import android.view.View;
import android.view.ViewGroup;

/**
 * Created by suzhenxi on 10/19/2016.
 */

public class ListAdapter2 extends BaseListFragmentAdapter {

    @Override
    public int getCount() {
        return 0;
    }

    @Override
    public Object getItem(int position) {
        return null;
    }

    @Override
    public long getItemId(int position) {
        return position;
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent) {
        if (convertView == null) {

        }

        return convertView;
    }
}

File: app/src/main/kotlin/suzp1984/github/io/exapidemo/app/fragments/ListAdapter2.kt
package suzp1984.github.io.exapidemo.app.fragments

import android.view.View
import android.view.ViewGroup

/**
 * Created by moses on 10/19/2016.
 */

class ListAdapter2 : BaseListFragmentAdapter() {

    override fun getCount(): Int {
        return 0
    }

    override fun getItem(position: Int): Any? {
        return null
    }

    override fun getItemId(position: Int): Long {
        return position.toLong()
    }

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View? {
        if (convertView == null) {

        }

        return convertView
    }
}
